### ISSUE TITLE
[Request] Web Interface for Virtual GPU Selection

### ISSUE THREAD

**Issue by BrainwreckedTech** [cockpit-project/cockpit-machines#554]

**Comment by BrainwreckedTech** [comment]
Whoopsie!  Yeah, I figured I'd seperate the two requests so one wasn't tied to the other.

**Comment by BrainwreckedTech** [comment]
The option could also be placed in the overview section as such, only showing the full set of tptions when the user brings up the edit dialog.

![mockup_virtual_gpus_overview](https://user-images.githubusercontent.com/3752022/152483670-ede49983-dced-4ecd-ade8-af767aa6d1b0.png)

**Comment by garrett** [comment]
Yeah, this makes more sense in the overview area, where we already have information about the CPU.

(The lists are more for things where someone can add N items, where N is somewhere between 0 and multiple. As far as vCPUs, it's just 1 setting, right?)

I'd suggest we should even combine all CPU information (CPU: host, 1 vCPU _edit_)

Also, why is CPU in overview and CPU is in Hypervisor details? It's a bit odd how this information is grouped, right? (We might want to rethink that.)
